site/_posts/2022-03-10-announcing-fusionauth-1-35.md

```diff
@@ -0,0 +1,100 @@
+
+Here's example code, which fetches a random Marvel movie quote from an unauthenticated API:
+
+```javascript
+var response = fetch("https://randommarvelquoteapi.herokuapp.com/", {
+                                  method: "GET",
+                                  headers: {
+                                    "Content-Type": "application/json"
+                                  }
+                                });
+```
+
+The request includes headers, so you may access any private APIs that are secured by an API key. You can make multiple requests to the same or different servers from within one lambda. You can also use other HTTP methods such as `POST`, allowing you to modify external databases during a lambda execution.
+
+The `response` object includes the body, status and headers. You can check to see if the API call succeeded and perform logic based on the response status. 
+
+Below we add the retrieved quote to the JWT claims, because who doesn't love a good movie quote?
+
+```javascript
+if (response.status === 200) {
+  var quoteBody = JSON.parse(response.body);
+  jwt.quote = quoteBody.quote;
+} else {
+  jwt.quote = "n/a";
+}
+```
+
+You can manage these lambdas via the administrative user interface, as below, or [the Lambda API](/docs/v1/tech/apis/lambdas).
+
+{% include _image.liquid src="/assets/img/blogs/release-1-35/api-requests-lambda.png" alt="Lambda which makes API requests." class="img-fluid" figure=true %}
+
+However, you must use the new GraalJS execution engine for this functionality.
+
+This feature is available for users on Essentials or Enterprise edition. Learn more about these editions by visiting [the pricing page](/pricing) or [contacting our sales team](/contact).
+
+## Internal upgrades
+
+1.35 and future versions of FusionAuth will ship with the GraalJS JavaScript execution engine. The sole JavaScript execution engine for previous versions was Nashorn, which has been [heading toward deprecation and removal from the JDK for a while](https://openjdk.java.net/jeps/335).
+
+GraalJS is a FusionAuth [tech preview feature](/docs/v1/tech/core-concepts/roadmap#tech-preview-features) at the moment. This means that, while the FusionAuth team will always strive for backward compatibility, there may be some changes required as we receive feedback from users and customers. In addition, GraalJS doesn't currently have the [performance optimizations of Nashorn](https://github.com/FusionAuth/fusionauth-issues/issues/571#issuecomment-1061614065), so when using it, benchmark your system to ensure it meets your needs.
+
+The benefits of GraalJS include the ability to use `fetch` as mentioned above and long-term support for a more modern version of JavaScript. While there is no internal timeline to remove Nashorn, it is recommended that you migrate your lambdas to GraalJS once it is out of tech preview.
+
+You can choose which engine to use on a per lambda basis:
```

Comment:
We could optionally mention that you can switch back and forth if you find something isn't working. But as you stated, to use `fetch` you must be on `GraalJS`. 

Another note is that our `fetch` isn't the actual `fetch` - it is `fetch` like, or `fetchy` if you will. :-) So folks shouldn't assume they can copy/paste any other fetch code they have previously written.